Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -473,12 +473,14 @@ void ChessWidget::input_engine_move()
         set_drag_enabled(false);
 
     set_override_cursor(Gfx::StandardCursor::Wait);
-    m_engine->get_best_move(board(), 4000, [this, drag_was_enabled](Chess::Move move) {
+    m_engine->get_best_move(board(), 4000, [this, drag_was_enabled](Optional<Chess::Move> move) {
```

Comment:
I've changed this to use `ErrorOr` instead. It's slightly strange that we construct an `Error` object whose value we don't use, but it makes it clear that we don't have a move because there was an error of some sort.